CONTRIBUTING.md

```diff
@@ -226,6 +226,28 @@ e.g. run tests against v 1.0
 
 If the "--api-version" argument is not specified, all runs act against default api version for each tested command.
 
+To run specific test:
+
+`pytest azext_iot/tests/central/test_iot_central_int.py::TestIotCentral::test_central_query_methods_run`
+
+Integration test environment variables needed:
+
+```
+env =
+    AZURE_TEST_RUN_LIVE=True
+    azext_iot_testrg=
+    azext_iot_teststorageaccount=
+    azext_iot_teststoragecontainer=
+    azext_iot_central_app_id=
+    azext_iot_central_scope_id=
+    azext_iot_central_token=
+    azext_iot_central_primarykey=
+    azext_iot_central_storage_cstring=
+    azext_iot_central_storage_container=
+```
+
+Not all of them are required. `azext_iot_teststoragecontainer` is the same as `azext_iot_central_storage_container`.
```

Comment:
why not incorporate this into the table above? We are also trying to reduce the number of pytest variables (unless you have a reason to have the storage account for central tests be different from the storage account for other iot services

Reply:
This is just an explanation for later contributors. That table above is for all integration tests here, not only for iot central. I didn't add anything new. I found that `azext_iot_teststoragecontainer` and `azext_iot_central_storage_container` are both working for iot central integration test so let others know.

Reply:
you can set them to the same value but by default they are different

Reply:
Thanks for the explanations. I removed all useless updates in this file.